### PR TITLE
Quiet Mode

### PR DESCRIPTION
**Pull request by joemillerr**
Changing permissions can get extremely noisey. This provides an option to suppress the logging from the change permissions function.

### CODE REVIEW

File: metadata.rb [hunk 1]
```diff
@@ -4,7 +4,7 @@
 maintainer_email 'itunixadminall@nordstrom.com'
 license          'Apache-2.0'
 description      'Recursively modify (chown, chmod)  owner, group, mode settings or delete files and directories'
-version          '1.1.7'
+version          '1.1.8'
```
Comment (MarkGibbons) [inline metadata.rb]: I'll make this 1.2.0.

**Comment by joemillerr** [comment]
@MarkGibbons Can we ignore this style issue?  Otherwise, what would you suggest?  Thanks!

File: libraries/change_help.rb [hunk 2]
```diff
@@ -78,7 +79,7 @@ def update(path)
   end
 
   def file_update(path, mode)
```
Comment (MarkGibbons) [inline libraries/change_help.rb]: Add to the delete logging also.

File: resources/default.rb [hunk 3]
```diff
@@ -38,7 +39,7 @@ def whyrun_supported?
 action :create do
   changed = update_files(new_resource.path, new_resource.pattern, new_resource.recursive, new_resource.follow_symlink,
                          new_resource.directory_mode, new_resource.file_mode, new_resource.group, new_resource.owner,
-                         new_resource.only_files, new_resource.only_directories, Chef::Config[:why_run])
+                         new_resource.quiet, new_resource.only_files, new_resource.only_directories, Chef::Config[:why_run])
```
Comment (MarkGibbons) [inline resources/default.rb]: Rubocop complaint.

**Comment by MarkGibbons** [comment]
Bump from 11 to 12.  The default is 5.

 Metrics/ParameterLists:
  Max: 11

File: resources/default.rb (hunk 3 above)
Comment (joemillerr) [reply resources/default.rb]: Works for me 👍 Thank you Mark!

**Comment by joemillerr** [comment]
@MarkGibbons should be all set now.  Thanks again!

**Pull request merged**